api/repositories/tx.go

```diff
@@ -90,12 +89,52 @@ func (r *txRepository) GetTxByHash(hash string) (*dto.RestTxResponse, error) {
 func (r *txRepository) GetTxCount() (*int64, error) {
 	var record db.Tracking
 
-	if err := r.db.Model(&db.Tracking{}).
+	err := r.db.Table(db.TableNameTracking).
```

Comment:
Not sure what we should use between db.Model or db.Table. Any thoughts. @WasinWatt @traviolus